pkg/controller/certificaterequests/ca/ca_test.go

```diff
@@ -438,10 +438,13 @@ func runTest(t *testing.T, test testT) {
 	}
 
 	controller := certificaterequests.New(apiutil.IssuerCA, ca)
-	controller.Register(test.builder.Context)
+	_, _, err := controller.Register(test.builder.Context)
+	if err != nil {
+		t.Errorf("controller.Register failed (%s)", err.Error())
```

Comment:
Nit: below on line 449 we use `%v` to print an error - perhaps would be nice to do it in the same way in both places (but probably not that important). Similar in other places in this PR.